pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "setuptools<60.0",
+    "setuptools<=67.3.2",
```

Comment:
I would suggest:
```suggestion
    "setuptools<68",
```
setuptools is pretty good about following [semver](https://semver.org/) and shouldn't break backwards compatibility without a major version bump.